.github/workflows/ci.yml

```diff
@@ -0,0 +1,65 @@
+---
+
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: "Testing"
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          # Recent Rubies and Rails
+          - ruby-version: '3.2'
+          - ruby-version: '3.1'
+          - ruby-version: '3.0'
+          - ruby-version: '2.7'
+          - ruby-version: '2.6'
+          - ruby-version: '2.6'
+          - ruby-version: '2.7'
+          - ruby-version: '2.6'
+          # Old Rubies and Rails
+          - ruby-version: '2.5'
+            bundler: '1'
+          - ruby-version: '2.4'
+            bundler: '1'
+          - ruby-version: '2.4'
+            bundler: '1'
+          # Failing with a stack trace in active support
+          # - ruby-version: '2.4'
+          #   rails-version: '4.1'
+          #   bundler: '1'
+
+    continue-on-error: "${{ endsWith(matrix.ruby-version, 'head') }}"
+
+    env:
+      CI: "1"
+
+    steps:
+      - name:                     "Checkout Code"
+        uses:                     "actions/checkout@v2"
+        timeout-minutes:          5
+        with:
+          fetch-depth:            0
+
+      # - name: Install required libs
+      #   run: |
+      #     sudo apt-get -yqq install libsqlite3-dev
+
+      - name: "Build Ruby"
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version:   "${{ matrix.ruby-version }}"
+          bundler:        "${{ matrix.bundler || 2 }}"
+          bundler-cache:  true
+        # env:
+        #   RAILS_VERSION: ${{ matrix.rails-version }}
```

Comment:
in case we decide to install certain versions of active support, this is an easy pattern for doing that